apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,14 +1,14 @@
 ack_generate_info:
-  build_date: "2021-06-22T20:41:21Z"
-  build_hash: 38e309d07369b1e33250473084f91db3894c4379
-  go_version: go1.16.4 darwin/amd64
-  version: v0.2.3
+  build_date: No Build Date Provided.
+  build_hash: No Git-hash Provided.
+  go_version: go1.14.14 darwin/amd64
+  version: (Unknown Version)
```

Comment:
@mbaijal why is this unknown

Reply:
@mbaijal You'll need to build the controllers from `code-generator` repository and not `community` one - i believe that we should remove build scripts from community repository soon.

Reply:
oh I see, thanks @A-Hilaly !